documents/data-sharing.md

```diff
@@ -0,0 +1,58 @@
+# Annual survey data sharing
+
+This document describes how data from the annual survey should be shared.
+
+## Objectives
+
+In priority order:
+
+* Protect the privacy of respondents.
+* Provide high quality data (unbiased data which is unlikely to be misinterpreted, c.f., raw data).
+* Provide useful and actionable data to the Rust project (i.e., teams and working groups, contributors, and the Foundation).
+* Provide useful data for users and potential users of Rust and providers of Rust tools and services.
+* Provide data in a timely manner.
+* Provide marketing for Rust.
+* Avoid overloading people working directly with the survey data (note that there were over 9000 completed surveys and multiple opportunities for free form responses).
+
+Non-goals:
+
```

Comment:
I believe an additional goal will not be to assist commercial products - e.g., if a company sells Rust based tooling, it is not a goal for the survey to explicitly support those projects with data.

Reply:
I think there is a bit of a fine line here. On the one hand, I don't think we should accept requests for custom stats, etc. from companies, and I don't think we should be producing statistics in general with the goal of helping companies sell more products. But, I think producing statistics that might be helpful for a person or team to persuade their management to resource tooling for Rust (commercial or not) seems like something which we should strive to do since it has obvious benefit for the community as a whole

Reply:
The key is broad applicability. We can take suggestions of what we should share from the community (including commercial products), but we should evaluate those suggestions on how applicable they are to the wider community. 

I believe we're on the same page here, so I think it's just that I believe this should be more explicitly laid out.